exercise/tex/exercise06.tex

```diff
@@ -149,13 +149,13 @@
     \end{equation}
     The voltage $\overline{u}_\mathrm{2}$ corresponds to $U_\mathrm{mot}$. Solving \eqref{eq:ex06t2_u2_gen} with respect to $\alpha$ results in
     \begin{equation} 
-        \alpha = \arccos( \frac{\overline{u}_\mathrm{2} \cdot p}{\hat{u}_\mathrm{1,LL}\pi \sin(\frac{p}{\pi})}).
+        \alpha = \arccos( \frac{\overline{u}_\mathrm{2} \cdot \pi}{\hat{u}_\mathrm{1,LL}\cdot p \cdot  \sin(\frac{\pi}{p})}).
     \end{equation}
     The following applies to the relevant cases:
     \begin{equation}     
         \begin{split}
-            \alpha_\mathrm{up} &= \arccos( \frac{\SI{466}{\volt} \cdot 6}{\SI{563}{\volt}\pi \sin(\frac{6}{\pi})})=\SI{30}{\degree}     \\
-            \alpha_\mathrm{down} &= \arccos( \frac{\SI{-466}{\volt} \cdot 6}{\SI{563}{\volt}\pi \sin(\frac{6}{\pi})})=\SI{150}{\degree}    \\
+            \alpha_\mathrm{up} &= \arccos( \frac{\SI{466}{\volt} \cdot \pi}{\SI{563}{\volt}\cdot 6 \cdot \sin(\frac{\pi}{6})})=\SI{30}{\degree} \\
+            \alpha_\mathrm{up} &= \arccos( \frac{\SI{-466}{\volt} \cdot \pi}{\SI{563}{\volt}\cdot 6 \cdot \sin(\frac{\pi}{6})})=\SI{150}{\degree} \\
```

Comment:
while the correction to the formula is correctly proposed, the firing angle label change towards "up" is incorrect as the calculated 150 degree fits to the 'down' case where lowering the load is associated with a negative secondary voltage at the motor terminals. In short, the label "\alpha_\mathrm{down}" should remain.